api/descriptions/policies/policies.md

```diff
@@ -1,10 +1,89 @@
-Manage your rules and policies programatically.
-
 Policies are sets of ordered rules.
 [Rule order](https://docs.twistlock.com/docs/latest/configure/rule_ordering_pattern_matching.html) determines how a policy is evaluated.
 
-In general, the policy endpoints don't have DELETE methods.
-Use the PUT method to delete all rules by submitting an empty JSON object.
+You can manage your rules and policies programmatically using the policy API endpoints.
+
+### How to Add / Update Policy Rules
+
+All of the `PUT /api/v1/policies/*` endpoints work similarly. 
+
+To add, edit, or remove vulnerability rules from a policy:
+
+1. Retrieve the entire policy, which includes all the vulnerability rules using the `GET` endpoint.
+
+  For example, the following cURL command uses basic auth to retrieve a list of all image vulnerability rules, pretty-prints the JSON response, and saves the results to a `vulnerability_rules.json` file.
+
+   ```bash
+   $ curl -k \
+     -u <USER> \
+     https://<CONSOLE>/api/v1/policies/runtime/host \
+     | jq '.' > vulnerability_rules.json
+   ```
+
+2. Modify the saved JSON with the updates, including any new rule insertions. **Note:** Rule order is important.
+
+3. Update the rules by pushing the new JSON payload into the `PUT` endpoint.
+
+   For example, the following cURL command installs the rules defined in your `vulnerability_rules.json` file.
+   
+   **Note:** Remember to specify the `@` symbol.
+
+   ```bash
+   $ curl -k \
+     -u <USER> \
+     -X PUT \
+     -H "Content-Type:application/json" \
+     https://<CONSOLE>/api/v1/policies/runtime/host \
+     --data-binary "@vulnerability_rules.json"
+   ```
+
+Any previously installed rules are overwritten.
+
+#### Minimum Rule Parameters
+
+To create or update a rule, the rule must specify the following:
```

Comment:
@KICKdesigns Although it might not be minimally required and enforced by the API, virtually every vuln rule will specify a non-default alert threshold. The default alert threshold of "Low" is way too broad and not actionable. Most customers will specify either a threshold of "Critical" or "High".

Reply:
@KICKdesigns I'm guessing your strategy is to specify the real min required params here, and then show an enhanced real-world cURL example with the alert threshold in the endpoint-specific descriptions. If that's the case, then I think there's nothing else to do here.

Reply:
@iansk I've added thresholds into the curl example.

Reply:
@KICKdesigns Can you please add some words about the thresholds here? Please also describe what thresholds your example curl command sets.

Reply:
@KICKdesigns Our API JSON should specify valid input and output ranges. When reviewing your curl example, I noticed no ranges are specified for `rules.alertThreshold.value` and `rules.blockThreshold.value` to specify Low, Medium, High, and Critical thresholds. I just created an issue with engineering for it.

If you find anything else like this, please note it in our shared spreadsheet.

Thanks!

Reply:
@iansk I've updated the doc to include the thresholds information. Let me know if there is anything else you want to add.

Reply:
@KICKdesigns LGTM, thanks